layout/plan.c

```diff
@@ -129,9 +174,16 @@ M0_INTERNAL struct m0_layout_plan * m0_layout_plan_build(struct m0_op *op)
 	if (rc != 0)
 		goto out;
 
+	m0_mutex_lock(&plan->lp_lock);
+
```

Comment:
What's the point of taking a lock that was just initialised? No other thread can see it yet, so no concurrency at this point.

Reply:
Sorry, it's a remnant from a previous version when the lock was taken inside of plop_alloc_init().

Dropped.

Reply:
It's because of plop_alloc_init() which checks that the lock is taken.

Reply:
Done.